sdk/core/azure-core/src/main/java/com/azure/core/implementation/AccessControllerUtils.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.implementation;
+
+import java.util.concurrent.Callable;
+import java.util.function.Supplier;
+
+/**
+ * Utility class for {@code AccessController} APIs.
+ * <p>
+ * This class exists to warden off usages of {@code java.security} APIs into a separate class that if it has loading
```

Comment:
```suggestion
 * This class exists to isolate usages of {@code java.security} APIs into a separate class that if it has loading
```

I think "warden" is not quite right here?